src/renderers/dom/client/utils/createMicrosoftUnsafeLocalFunction.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule createMicrosoftUnsafeLocalFunction
+ */
+
+/* globals MSApp */
+
+'use strict';
+
+/**
+ * Create a function which has 'unsafe' privileges (required by windows8 apps)
+ */
+var createMicrosoftUnsafeLocalFunction = function(func) {
+  if (typeof MSApp !== 'undefined' && MSApp.execUnsafeLocalFunction) {
+    return function(arg0, arg1, arg2, arg3) {
+      MSApp.execUnsafeLocalFunction(function() {
+        return func(arg0, arg1, arg2, arg3);
```

Comment:
I was told (a while ago) by Sebastian to prefer multiple explicit args rather than using the `arguments` variable, because it avoids an array allocation, and jsperf (http://jsperf.com/asdfewgerge/2) agrees with him.